integration/raft/channel_participation_test.go

```diff
@@ -123,59 +126,101 @@ var _ = Describe("ChannelParticipation", func() {
 				Height:          1,
 			}
 
-			for _, o := range orderers {
+			for _, o := range members {
 				By("joining " + o.Name + " to channel as a member")
 				channelparticipation.Join(network, o, "participation-trophy", genesisBlock, expectedChannelInfoPT)
-				channelparticipation.ListOne(network, o, expectedChannelInfoPT)
+				channelInfo := channelparticipation.ListOne(network, o, "participation-trophy")
+				Expect(channelInfo).To(Equal(expectedChannelInfoPT))
 			}
 
 			By("waiting for the leader to be ready")
-			leader := findLeader(ordererRunners)
+			memberRunners := []*ginkgomon.Runner{ordererRunners[0], ordererRunners[1]}
+			leader := findLeader(memberRunners)
 
-			By("ensuring the channel is usable by submitting a transaction to each orderer")
+			By("ensuring the channel is usable by submitting a transaction to each member")
 			env := CreateBroadcastEnvelope(network, peer, "participation-trophy", []byte("hello"))
-			for _, o := range orderers {
+			for _, o := range members {
 				By("submitting transaction to " + o.Name)
 				expectedChannelInfoPT.Height++
 				resp, err := ordererclient.Broadcast(network, o, env)
 				Expect(err).NotTo(HaveOccurred())
 				Expect(resp.Status).To(Equal(common.Status_SUCCESS))
 				expectedBlockNumPerChannel := map[string]int{"participation-trophy": int(expectedChannelInfoPT.Height - 1)}
-				assertBlockReception(expectedBlockNumPerChannel, orderers, peer, network)
+				assertBlockReception(expectedBlockNumPerChannel, members, peer, network)
 
 				By("checking the channel height")
-				channelparticipation.ListOne(network, o, expectedChannelInfoPT)
+				channelInfo := channelparticipation.ListOne(network, o, "participation-trophy")
+				Expect(channelInfo).To(Equal(expectedChannelInfoPT))
 			}
 
+			By("joining orderer3 to the channel as a follower")
+			// make sure we can join using a config block from one of the other orderers
+			configBlockPT := nwo.GetConfigBlock(network, peer, orderer2, "participation-trophy")
+			expectedChannelInfoPTFollower := channelparticipation.ChannelInfo{
+				Name:            "participation-trophy",
+				URL:             "/participation/v1/channels/participation-trophy",
+				Status:          "onboarding",
+				ClusterRelation: "follower",
+				Height:          0,
+			}
+			channelparticipation.Join(network, orderer3, "participation-trophy", configBlockPT, expectedChannelInfoPTFollower)
+			channelInfo := channelparticipation.ListOne(network, orderer3, "participation-trophy")
+			Expect(channelInfo).To(Equal(expectedChannelInfoPTFollower))
+
```

Comment:
I think this may generate a race on the height and status. If the REST query is faster than the follower pulling the first block, no problem. However if the REST query is slow and the follower is fast, we'll get a mismatch, as the height will be >0. By the same logic, if the follower is really fast, it will finish onboarding....

Reply:
Agreed. Removed this check as it's unnecessary and not guaranteed to be equal.